### DISCUSSION TITLE
JS SDK Game Show and Tell

### DISCUSSION THREAD

**Discussion by BenKurrek** [near/near-sdk-js#30] [Show and tell]
I have pushed my game to github and wanted to get some feedback as to the functionality / actual code. If there’s stuff that I could be doing differently or am doing wrong, please let me know. This is my first project using the JS SDK and would love to know if I did things correctly or incorrectly.

https://github.com/BenKurrek/near-sdk-js-game/blob/main/readme.md

**Comment by volovyks** [comment]
@BenKurrek today it is hard to say since `near-sdk-js` setup has been changed. Lots of new APIs have been added.
Let's update it or resolve this conversation,

**Reply by volovyks** [reply]
@BenKurrek are you planning to add contract tests to the project (with Workspaces JS). I wanted to contribute, and refactor something, but I'm afraid to break all the logic :)

**Reply by BenKurrek** [reply]
Unfortunately I wasn't planning on it because I work with an M1 mac and workspaces aren't supported for me..

**Reply by volovyks** [reply]
Oh.. I hope we will fix that soon.
